src/Build.OM.UnitTests/Definition/ProjectItem_Tests.cs

```diff
@@ -2158,32 +2158,64 @@ public void RemoveWithItemReferenceOnMatchingMetadata()
                 <I2 Include='c2' M1='3' m2='Y'/>
                 <I2 Include='d2' M1='y' m2='d'/>
 
-                <I2 Remove='@(I1)' MatchOnMetadata='M1;m2'/>");
+                <I2 Remove='@(I1)' MatchOnMetadata='M1'/>");
```

Comment:
Change to `MatchOnMetadata='m1` to assert that metadata name match is case insensitive.